ext/bcmath/bcmath.c

```diff
@@ -434,6 +434,68 @@ PHP_FUNCTION(bcmod)
 }
 /* }}} */
 
+PHP_FUNCTION(bcdivmod)
+{
+	zend_string *left, *right;
+	zend_long scale_param;
+	bool scale_param_is_null = 1;
+	bc_num first = NULL, second = NULL, quot, rem;
+	int scale = BCG(bc_precision);
+
+	ZEND_PARSE_PARAMETERS_START(2, 3)
+		Z_PARAM_STR(left)
+		Z_PARAM_STR(right)
+		Z_PARAM_OPTIONAL
+		Z_PARAM_LONG_OR_NULL(scale_param, scale_param_is_null)
+	ZEND_PARSE_PARAMETERS_END();
+
+	if (scale_param_is_null) {
+		scale = BCG(bc_precision);
+	} else if (scale_param < 0 || scale_param > INT_MAX) {
+		zend_argument_value_error(3, "must be between 0 and %d", INT_MAX);
+		RETURN_THROWS();
+	} else {
+		scale = (int) scale_param;
+	}
+
+	BC_ARENA_SETUP;
+
+	bc_init_num(&quot);
```

Comment:
Why do you initialize `quot` and `rem` like this instead of setting them to NULL at the declaration site?